Article/Article.js

```diff
@@ -1,5 +1,5 @@
 /* Using jQuery, select all the article elements */
-let articles;
+let articles = $('.article');
```

Comment:
Well done so far. Using the jQuery selector to assign articles, so far it is in its state of object/mock array.